### PR TITLE
CRI-O: add `pull-kubernetes-node-crio-cgrpv2-imagevolume-e2e` test job

### PR DESCRIPTION
**Pull request by saschagrunert**
Adding a new job for the imagevolume alpha tests.

Requires https://github.com/cri-o/cri-o/pull/8408
Will be required for https://github.com/kubernetes/kubernetes/pull/126220

### CODE REVIEW

File: jobs/e2e_node/crio/templates/generate [hunk 1]
```diff
@@ -28,7 +28,7 @@ declare -A CONFIGURATIONS=(
     ["crio_cgroupsv1"]="root cgroups-v1 criu-enabled"
     ["crio_cgroupsv1_eventedpleg"]="root cgroups-v1 eventedpleg"
     ["crio_cgroupsv1_hugepages"]="root cgroups-v1 hugepages"
-    ["crio_cgroupsv2"]="root"
+    ["crio_cgroupsv2"]="root-v2"
```
Comment (saschagrunert) [inline jobs/e2e_node/crio/templates/generate:31]: I'll swap the other tests as follow-up if everything is green.

**Comment by saschagrunert** [comment]
/hold

Ready for review, I just like to hold to have control over when it merges.

File: jobs/e2e_node/crio/templates/crio_cgroupsv2.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ storage:
       mode: 0644
     - path: /etc/crio/crio.conf.d/20-crio.conf
       contents:
-        local: 20-crio.conf
+        local: 20-crio-v2.conf
```
Comment (haircommander) [inline jobs/e2e_node/crio/templates/crio_cgroupsv2.yaml:19]: what's the diff between these two? do we need a separate one if we're only updating the commit below?

**Comment by saschagrunert** [comment]
Asking for node approval @yujuhong @Random-Liu @dchen1107 @derekwaynecarr @sjenning @mrunalp @klueska @SergeyKanzhelev @endocrimes

File: jobs/e2e_node/crio/templates/crio_cgroupsv2.yaml (hunk 2 above)
Comment (saschagrunert) [reply jobs/e2e_node/crio/templates/crio_cgroupsv2.yaml:19]: The libexec path usage is new.

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm
/approve

**Pull request merged**